### DISCUSSION TITLE
Feature freeze in preparation for release 0.30.0

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#1619] [Announcements]
Can we call a feature freeze and focus on getting everything in that is on [the list](https://github.com/rust-bitcoin/rust-bitcoin/discussions/1501)?

I've worked through all the issues labelled "API hole" and/or "milestone 0.30.0". I've labelled PRs that fix such issues as "P-High". The few that are not done have comments asking if they can have the label/milestone removed.

IIRC February was the target month for release so this gives us a month to review and merge all the open PRs.

We could let #1555 through too if @Kixunil  wants that one in :)

**Comment by Kixunil** [comment]
Oh, crap, I wanted so many more things in... But OK, I guess. The new version will be very helpful for my project anyway. Which makes me think, could we get `Weight` and `FeeRate` newtypes in as well as changing `TxOut::value` to `Amount`? It'd help greatly and I'm willing to prioritize to make PRs for these.

#1555 and #1186  would be nice too to minimize conflicts

**Reply by apoelstra** [reply]
Let's get this release out :) I think to get your "so many more things in" let's start doing pre-releases every month, as suggested in #1520, so that at least your new stuff will be available on crates, even if it's not in an official release.
